docs/shipping/Security/one-password.md

```diff
@@ -0,0 +1,94 @@
+---
+id: 1Password
+title: 1Password
+overview: 1Password is a password manager. This integration allows you to send event logs to your Logz.io account.
+product: ['logs','siem']
+os: ['windows', 'linux']
+filters: ['Security']
+logo: https://logzbucket.s3.eu-west-1.amazonaws.com/logz-docs/shipper-logos/1password.png
+logs_dashboards: []
+logs_alerts: []
+logs2metrics: []
+metrics_dashboards: []
+metrics_alerts: []
+drop_filter: []
+---
+
+You can ship logs available from the 1Password API with Logzio-api-fetcher.
+
+## Pull Docker Image
+Download the logzio-api-fetcher image:
+
+```shell
+docker pull logzio/logzio-api-fetcher
+```
+
+## Configuration
+Create a local config file `config.yaml`.  
+
+```yaml
+apis:
+  - name: 1Password test
+    type: 1password
+    onepassword_bearer_token: <<1PASSWORD_BEARER_TOKEN>>
+    url: https://events.1password.com/api/v1/auditevents
+    method: POST
+    days_back_fetch: 7
+    scrape_interval: 5
+    additional_fields:
+      type: 1password
+
+logzio:
+  url: https://<<LISTENER-HOST>>:8071
+  token: <<SHIPPING_TOKEN>>
+```
+
+### 1Password configuration options
+| Parameter Name           | Description                                                                                     | Required/Optional | Default           |
+|--------------------------|-------------------------------------------------------------------------------------------------|-------------------|-------------------|
+| name                     | Name of the API (custom name)                                                                   | Optional          | the defined `url` |
+| onepassword_bearer_token | The 1Password Bearer token                                                                      | Required          | -                 |
+| url                      | The request URL                                                                                 | Required          | -                 |
+| method                   | The request method (`GET` or `POST`)                                                            | Optional          | `GET`             |
+| additional_fields        | Additional custom fields to add to the logs before sending to logzio                            | Optional          | -                 |
+| days_back_fetch          | The amount of days to fetch back in the first request. Applies a filter on `since` parameter.   | Optional          | -                 |
```

Comment:
... on the `since` parameter? @8naama

Reply:
clarified it in new commit 🙏🏼  it's the name of date filter 1password api uses to decide from when  to return results from